packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2710,7 +2738,7 @@ export class ContainerRuntime
 
 			this.validateAndProcessRuntimeMessage(messageWithContext, localOpMetadata);
 
-			this.emit("op", message, messageWithContext.modernRuntimeMessage);
+			this.emit("op", message, messageWithContext.isRuntimeMessage);
```

Comment:
Add a changeset for this.....? Maybe?  Or that may just cause unnecessary confusion.  Up to you.

Reply:
Not adding a changeset, I think the value now is more accurate than it was previously?